src/Chainweb/Pact/PactService.hs

```diff
@@ -971,7 +971,7 @@ execValidateBlock currHeader plData = do
     isGenesisBlock = isGenesisBlockHeader currHeader
 
 validateSilenceDates :: MonadThrow m => BlockHeader -> PayloadData -> m ()
-validateSilenceDates bh plData = case txSilenceEndDate (_blockChainwebVersion bh) of
+validateSilenceDates bh plData = case txActivationDate (_blockChainwebVersion bh) of
```

Comment:
```suggestion
validateSilenceDates bh plData = case txSilenceEndDate (_blockChainwebVersion bh) of
```

We must not change this existing validation rule.

Reply:
We must, or else transactions from Dec 5 to Dec 6 will be allowed.